xonsh/tools.py

```diff
@@ -127,6 +127,11 @@ def _expandpath(path):
     return expand_path(path, expand_user=expand_user)
 
 
+def random_choice(lst):
+    """Returns random element from the list"""
+    return lst[datetime.datetime.now().microsecond % len(lst)]
```

Comment:
What if `len(lst)` is greater than the size of microsecond?  It seems like there should be some kind of warning about this.

Reply:
done